package.json

```diff
@@ -42,6 +43,6 @@
   },
   "main": "d2l-colors.js",
   "dependencies": {
-    "@polymer/polymer": "^3.0.0"
+    "@brightspace-ui/core": "^0.2"
```

Comment:
I think Polymer needs to stay as a non-dev dependency now that the deprecated colors are still there.

Reply:
Thanks.  I forgot I'd moved that before.